control-plane/rest/openapi-specs/v0_api_spec.yaml

```diff
@@ -1131,6 +1131,12 @@ paths:
       tags:
         - Pools
       operationId: get_pools
+      parameters:
+        - in: query
```

Comment:
btw if this is a query, then not finding any pools is valid, should get empty vector rather than error
Please check if this is the case

Reply:
It returns an empty vector

![image](https://github.com/openebs/mayastor-control-plane/assets/13465383/7fc1eaf1-8f89-4df6-a603-e11a1237575b)